### PR TITLE
Appveyor

### PR DESCRIPTION
**Pull request by jlaura**

### CODE REVIEW

**Comment by jlaura** [comment]
I am done with this for now:

- Linux still passing
- Win passing as long as we don't upload (fine here, because we use Conda-forge and I don't feel the need to push a 'dev' version of plio to conda)
- Mac queued, but I don't know why it wouldn't pass sometime in the night

This 'should' get all CI passing. No reason that #114 needs to wait on this though!

File: environment.yml [hunk 1]
```diff
@@ -3,8 +3,8 @@ channels:
   - conda-forge
   - usgs-astrogeology
 dependencies:
-  - libgdal 
-  - gdal 
+  - libgdal < 3 
+  - gdal < 3
```
Comment (Kelvinrr) [inline environment.yml]: lmoa

File: plio/io/tests/test_metadata.py [hunk 2]
```diff
@@ -52,7 +52,8 @@ def test_export_to_proj4(srs_mars):
     Check that proj4 is not supporting Moon2000_Mercator
     """
     proj4 = srs_mars.ExportToProj4()
-    assert proj4 == '+proj=merc +lon_0=180 +lat_ts=0 +x_0=0 +y_0=0 +a=1737400 +b=1737400 +units=m +no_defs '
+    for element in '+proj=merc +lon_0=180 +lat_ts=0 +x_0=0 +y_0=0 +a=1737400 +b=1737400 +units=m +no_defs'.split():
+        assert element in proj4
```
Comment (Kelvinrr) [inline plio/io/tests/test_metadata.py]: :+1:

File: plio/io/io_gdal.py [hunk 3]
```diff
@@ -191,6 +191,7 @@ def geotransform(self):
 
     @property
     def forward_affine(self):
+        print(self.geotransform)
```
Comment (Kelvinrr) [inline plio/io/io_gdal.py]: print statement left in

File: plio/io/tests/test_io_gdal.py [hunk 4]
```diff
@@ -62,7 +62,9 @@ def test_latlon_corners(self):
     def test_spheroid(self):
         sphere = self.dataset.spheroid
         self.assertAlmostEqual(sphere[0], 3396190.0, 6)
-        self.assertEqual(self.dataset.spheroid, (3396190.0, 3376200.0, 169.8944472236118))
+        self.assertAlmostEqual(self.dataset.spheroid[0], 3396190.0)
+        self.assertAlmostEqual(self.dataset.spheroid[1], 3376200.0)
+        self.assertAlmostEqual(self.dataset.spheroid[2], 169.8944472236118)
         self.assertAlmostEqual(sphere[1], 3376200.0, 6)
```
Comment (acpaquette) [inline plio/io/tests/test_io_gdal.py]: Any particular reason this needed to be unpacked?

**Comment by acpaquette** [comment]
Looks fine overall :+1:

**Pull request merged**